pages/page.py

```diff
@@ -34,10 +33,7 @@ def __init__(self, testsetup):
     def is_the_current_page(self):
         if self._page_title:
             WebDriverWait(self.selenium, 10).until(lambda s: self.selenium.title)
-
-        Assert.equal(self.selenium.title, self._page_title,
-                     'Expected page title: %s. Actual page title: %s' % (
-                         self._page_title, self.selenium.title))
+        assert self.selenium.title == self._page_title
```

Comment:
I believe we were going to try to stick with `expected == actual` but this looks like the opposite.